health/notifications/health_alarm_notify.conf

```diff
@@ -278,6 +278,13 @@ STACKPULSE_WEBHOOK=""
 
 DEFAULT_RECIPIENT_STACKPULSE=""
 
+#------------------------------------------------------------------------------
+# opsgenie global notification options
+SEND_OPSGENIE="YES"
+
+# Api key
+OPSGENIE_API_KEY=""
+
```

Comment:
Don't you need to define `DEFAULT_RECIPIENT_OPSGENIE` here?

Reply:
Opsgenie will never use this, but I agree we need to have one. I brought it with the last commit.